netcore/Makefile

```diff
@@ -5,35 +5,38 @@ include ../mcs/build/config.make
 #
 
 # Extracted MicrosoftPrivateCoreFxNETCoreAppVersion from https://github.com/dotnet/coreclr/blob/master/eng/Versions.props#L18
-NETCORETESTS_VERSION := 4.6.0-preview6.19271.5
+NETCORETESTS_VERSION := 4.6.0-preview6.19273.8
 
 # Extracted MicrosoftNETCoreAppVersion from https://github.com/dotnet/coreclr/blob/master/eng/Versions.props#L22
-NETCOREAPP_VERSION := 3.0.0-preview6-27720-09
+NETCOREAPP_VERSION := 3.0.0-preview6-27727-02
 
 # Extracted from https://dotnetcli.blob.core.windows.net/dotnet/aspnetcore/Runtime/master/latest.version
 ASPNETCOREAPP_VERSION := 3.0.0-preview-18614-0151
 
-include roslyn.make
+ROSLYN_VERSION := 3.2.0-beta3-19279-03
 
 ifeq ($(HOST_PLATFORM),win32)
 NETCORESDK_EXT = zip
 UNZIPCMD = python -c "import zipfile,sys; zipfile.ZipFile(sys.argv[1], 'r').extractall()"
 XUNIT_FLAGS =
 TESTS_PLATFORM = Windows_NT.x64
+DOTNET = $(shell .\init-tools.ps1)
```

Comment:
Make shells out to _bash_, so this should be in a format which MSYS2 bash will work with (e.g. maybe just `./init-tools.ps1`, otherwise the full path to `/foo/bar/powershell init-tools.ps1`?)

Right now it fails with:
```
make: Entering directory 'D:/a/1/s/netcore'
make: Makefile:74: pipe: No such file or directory
restore roslyn-restore.csproj -p:RoslynVersion=3.2.0-beta3-19279-03 --packages roslyn/packages -p:OutputPath=roslyn/restore/
restore: Unknown option -p:RoslynVersion=3.2.0-beta3-19279-03. Try restore --help for more info.
make: *** [Makefile:74: update-roslyn] Error 1
```

Reply:
Windows support is not finished yet, `init-tools.ps1` is not even commited 🙂 but it all should work on macOS/Linux

Reply:
Yes, OSX and Linux are building.

Reply:
Windows should now work too.